<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::PropertyIntPairEditor</class>
 <widget class="QWidget" name="GammaRay::PropertyIntPairEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSpinBox" name="xBox">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="yBox">
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
